# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kde/koffice/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- koffice-1.3.1/karbon/widgets/vcanvas.cc.orig	2004-04-12 11:18:46.000000000 +0200
+++ koffice-1.3.1/karbon/widgets/vcanvas.cc	2004-05-10 10:24:47.550169576 +0200
@@ -306,12 +306,12 @@
 
 /// repaints just a rect area (no scrolling)
 void
-VCanvas::repaintAll( const KoRect & )
+VCanvas::repaintAll( const KoRect &  r)
 {
 	//if( m_view->layersDocker() )
 //		m_view->layersDocker()->updatePreviews();
 	VPainter *p = m_view->painterFactory()->painter();
-	KoRect rect( rect().x(), rect().y(), rect().width(), rect().height() );
+	KoRect rect( r.x(), r.y(), r.width(), r.height() );
 	p->blit( rect );
 
 	// draw handle:
